bottlecap/LICENSE-3rdparty.yml

```diff
@@ -21481,46 +21481,7 @@ third_party_libraries:
 
       END OF TERMS AND CONDITIONS
   - license: Unicode-3.0
-    text: |
-      UNICODE LICENSE V3
-
-      COPYRIGHT AND PERMISSION NOTICE
-
-      Copyright © 1991-2023 Unicode, Inc.
-
-      NOTICE TO USER: Carefully read the following legal agreement. BY
-      DOWNLOADING, INSTALLING, COPYING OR OTHERWISE USING DATA FILES, AND/OR
-      SOFTWARE, YOU UNEQUIVOCALLY ACCEPT, AND AGREE TO BE BOUND BY, ALL OF THE
-      TERMS AND CONDITIONS OF THIS AGREEMENT. IF YOU DO NOT AGREE, DO NOT
-      DOWNLOAD, INSTALL, COPY, DISTRIBUTE OR USE THE DATA FILES OR SOFTWARE.
-
-      Permission is hereby granted, free of charge, to any person obtaining a
-      copy of data files and any associated documentation (the "Data Files") or
-      software and any associated documentation (the "Software") to deal in the
-      Data Files or Software without restriction, including without limitation
-      the rights to use, copy, modify, merge, publish, distribute, and/or sell
-      copies of the Data Files or Software, and to permit persons to whom the
-      Data Files or Software are furnished to do so, provided that either (a)
-      this copyright and permission notice appear with all copies of the Data
-      Files or Software, or (b) this copyright and permission notice appear in
-      associated Documentation.
-
-      THE DATA FILES AND SOFTWARE ARE PROVIDED "AS IS", WITHOUT WARRANTY OF ANY
-      KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-      MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT OF
-      THIRD PARTY RIGHTS.
-
-      IN NO EVENT SHALL THE COPYRIGHT HOLDER OR HOLDERS INCLUDED IN THIS NOTICE
-      BE LIABLE FOR ANY CLAIM, OR ANY SPECIAL INDIRECT OR CONSEQUENTIAL DAMAGES,
-      OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
-      WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
-      ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THE DATA
-      FILES OR SOFTWARE.
-
-      Except as contained in this notice, the name of a copyright holder shall
-      not be used in advertising or otherwise to promote the sale, use or other
-      dealings in these Data Files or Software without prior written
-      authorization of the copyright holder.
+    text: NOT FOUND
```

Comment:
Removing a license 🤔 

This bug has been happening for a while, sometimes it gets fixed randomly

Reply:
The bug is that our CI is installing cargo-bundle-licenses via an unqualified installation command, so it was pulling the latest version, in this case v3.

My local machine had v2.x, so there were some conflicts. Our cache key is perhaps a big ambiguous, but in this case it's passing and the fix was just updating `cargo-bundle-licenses` on my machine.